kamailio/default.cfg

```diff
@@ -87,10 +91,11 @@ disable_sctp = yes
 mpath="/usr/lib64/kamailio/modules/:/usr/lib/x86_64-linux-gnu/kamailio/modules/"
 
 ######## Kamailio stun module ########
+loadmodule "outbound.so"
 loadmodule "stun.so"
 
 ######## Kamailio outbound module ########
```

Comment:
@lazedo for consistency can you just move this whole section above line 93 ("######## Kamailio stun module ########")?